environment.yml

```diff
@@ -1,6 +1,7 @@
 name: morphometrics
 dependencies:
-  - python=3.9.0
+  - python=3.9
+  - conda-forge::pyqt
```

Comment:
It was the pymeshlab dependency we needed this for, is that right?

Reply:
Yep!